crest/Assets/Crest/Crest/Shaders/OceanHelpersNew.hlsl

```diff
@@ -21,6 +21,18 @@ float3 WorldToUV(in float2 i_samplePos, in CascadeParams i_cascadeParams, in flo
 	return float3(uv, i_sliceIndex);
 }
 
+float3 WorldToUVSource(in float2 i_samplePos, in CascadeParams i_cascadeParams, in float i_sliceIndex)
```

Comment:
i didnt automatically understand the reason for this function. is there an iteration on the naming that would make it clearer (even if its quite explicit), or could we add a comment above it. just to make it clear which function to use from this library for different situations.

Reply:
Removed as fixed in C#.